wakeup-server/tests/test_interactive_devices_exception.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+
+def test_empty_name(client):
+    response = client.post("/interactive_devices", json={"name": "", "type": "SOUND"})
+    obj = response.get_json()
+    with pytest.raises(TypeError) as e:
+        name = obj['name']
+    msg = e.value.args[0]
+    assert 'NoneType' in msg
+
+
+def test_too_long_name(client):
```

Comment:
Please leave a #TODO comment with a little sentence to say that this is still need to be implemented in the server if it is not implemented yet

Reply:
Same applies for other tests